File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/report/ParticipareReprezentantiArbLaActiuniInAfaraAsociatieiReportRowModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.report;

import java.io.Serializable;
import java.util.Date;

public class ParticipareReprezentantiArbLaActiuniInAfaraAsociatieiReportRowModel implements Serializable{

	private Date data;
	private String actiune;
	private String subiectAgenda;
	private String detaliuSubiectAgenda;
	private String participanti;

	public Date getData() {
		return data;
	}

	public void setData(Date data) {
		this.data = data;
	}

	public String getActiune() {
		return actiune;
	}

	public void setActiune(String actiune) {
		this.actiune = actiune;
	}

	public String getParticipanti() {
		return participanti;
	}

	public void setParticipanti(String participanti) {
		this.participanti = participanti;
	}

	public String getSubiectAgenda() {
		return subiectAgenda;
	}

	public void setSubiectAgenda(String subiectAgenda) {
		this.subiectAgenda = subiectAgenda;
	}

	public String getDetaliuSubiectAgenda() {
		return detaliuSubiectAgenda;
	}

	public void setDetaliuSubiectAgenda(String detaliuSubiectAgenda) {
		this.detaliuSubiectAgenda = detaliuSubiectAgenda;
	}
}

File: CloudFlow/FrontEnd-App/src/app/shared/model/reports/participare-reprezentanti-arb-la-actiuni-in-afara-asociatiei-report-row.model.ts
import { JsonObject, JsonProperty } from "json2typescript";
import { JsonDateConverter } from "../../json-mapper";

@JsonObject
export class ParticipareReprezentantiArbLaActiuniInAfaraAsociatieiReportRowModel {

	@JsonProperty("data", JsonDateConverter)
	public data: Date = null;

	@JsonProperty("actiune", String)
	public actiune: string = null;
    
	@JsonProperty("subiectAgenda", String)
	public subiectAgenda: string = null;
    
	@JsonProperty("detaliuSubiectAgenda", String)
	public detaliuSubiectAgenda: string = null;
    
	@JsonProperty("participanti", String)
	public participanti: string = null;
}